docs/source/models/gbd2017/causes/lower_respiratory_infections/index.rst

```diff
@@ -92,7 +117,7 @@ Data Description
      -
    * - I
      - disability weights
-     - disability_weight_s670 :math:`\times` prevalence_s670+ disability_weight_s669 :math:`\times` prevalence_s669
+     - disability_weight_s670 :math:`\times` prevalence_s670+ disability_weight_s669 :math:`\times` prevalence_s669 + disability_weight_s671 :math:`\times` prevalence_s671
      - GBD assumes 15% of LRI cases as severe and 85% as moderate [GBD-2017-YLD-Capstone-Appendix-1]_.
```

Comment:
this currently links to the measles page

Reply:
In response to @NathanielBlairStahn 's comment--from the YLD appendix, it sounds like they did a systematic review to determine that LRI cases split into 85/15 moderate/severe. If GB is one of the sequelae you get when you pull GB sequelae, then I believe that that would've been redistributed to GB during como, and after the initial severity split calculations, to result in those three sequelae (moderate LRI, severe LRI, GB).

Reply:
We can verify by pulling the prevalence of those sequelae!

Reply:
> this currently links to the measles page

This is a systematic problem we need to figure out how to fix: Sphinx is written so that citations have to be unique across the entire documentation, not just within a page. So in order to get citations to work correctly, each writer will have to be responsible for coming up with unique citation abbreviations for their references, which seems a bit burdensome, though we could try to come up with a standard naming convention to help facilitate this.

Other options would be to see if we can reconfigure Sphinx (@collijk said he would look into this), or to come up with a totally different system for doing citations.  Any suggestions would be welcome.

Reply:
> In response to @NathanielBlairStahn 's comment--from the YLD appendix, it sounds like they did a systematic review to determine that LRI cases split into 85/15 moderate/severe. If GB is one of the sequelae you get when you pull GB sequelae, then I believe that that would've been redistributed to GB during como, and after the initial severity split calculations, to result in those three sequelae (moderate LRI, severe LRI, GB).

So, assuming the 3 sequelae are mutually exclusive and their prevalences add up to 100% of LRI, does that mean essentially that people with the GB sequela can be assumed to also have an 85/15 moderate/severe split for the severity of LRI they experience?

Is it possible to pull GB sequelae directly, since it's an impairment not a cause (i.e. as opposed to searching for all the causes that list GB as one of the sequelae)? I guess I'm confused about how impairments work and how COMO works. What does it mean that "that would've been redistributed to GB during como"?